ksp_plugin_adapter/localization_extensions.cs

```diff
@@ -28,23 +29,51 @@ internal static class L10N {
     return Localizer.Format("#Principia_GrammaticalForm_Standalone", name);
   }
 
-  public static string NameWithoutArticle(this CelestialBody body) {
-    // This will need to be adjusted when we add support for other languages
-    // with articles.
-    return (Localizer.CurrentLanguage == english_us_ &&
-            body.displayName.StartsWith("The ")) ? body.name : body.displayName;
+  public static string StandaloneName(this CelestialBody celestial) {
+    return Standalone(celestial.NameWithoutArticle());
+  }
+
+  private static bool StartsWithCapitalizedDefiniteArticle(string s) {
+    return (Localizer.CurrentLanguage == english_us_ && s.StartsWith("The ")) ||
+           (Localizer.CurrentLanguage == french_ &&
+            (s.StartsWith("La ") || s.StartsWith("Le ")));
```

Comment:
What about `L'` or `Les`?

Reply:
This is there to work around stock because we must work around stock ; other mods should use `ornithorhynque^d`, not `L’ornithorhynque` (and should we handle APOSTROPHE or only RIGHT SINGLE QUOTATION MARK? if we go that way it never ends, we might want to handle the lower case as well…).

Stock only has `La Mune` and `Le Soleil`.